src/components/Settings/SettingsTab.tsx

```diff
@@ -53,12 +65,28 @@ export const SettingsTab = ({
   isChatDisabled,
   setIsChatDisabled,
   clearChat,
+  roomTitle,
+  roomDescription,
+  roomTitleColor,
 }: SettingsTabProps) => {
   const [updateTS, setUpdateTS] = useState(0);
   const [permModalOpen, setPermModalOpen] = useState(false);
   const [validVanity, setValidVanity] = useState(true);
   const [validVanityLoading, setValidVanityLoading] = useState(false);
   const [adminSettingsChanged, setAdminSettingsChanged] = useState(false);
+  const [_roomTitle, _setRoomTitle] = useState<string | undefined>('');
```

Comment:
why the _ prefix?

Reply:
I guess I could raname it to something like roomTitleInput